packages/template-retail-react-app/app/components/swatch-group/swatch.jsx

```diff
@@ -14,6 +14,7 @@ import {
     useMultiStyleConfig
 } from '@salesforce/retail-react-app/app/components/shared/ui'
 import {Link as RouteLink} from 'react-router-dom'
+import {useBreakpointValue} from '@salesforce/retail-react-app/app/components/shared/ui'
 
 /**
  * The Swatch Component displays item inside `SwatchGroup`. For proper keyboard accessibility,
```

Comment:
I believe you had done some work for accessibility in this PR. How should we test-drive this change? Thanks.

Reply:
I re-implemented the existing logic we had to be more react-y. It didn't look like there were any tests for it tho. I've added some basic tests for using arrow keys to select swatches.

Reply:
Oh I meant manual tests.. how should we manually test the accessibility changes? I'm not familiar with how we did that last time.